app/services/apply_api/import_application.rb

```diff
@@ -4,16 +4,21 @@ module ApplyApi
   class ImportApplication
     include ServicePattern
 
+    class ApplyApiMissingDataError < StandardError; end
+
     def initialize(application_data:)
       @application_data = application_data
     end
 
     def call
       return unless provider
+      return if provider_a_hei?
 
       application.update!(application: application_data.to_json, provider: provider)
 
       application
+    rescue NoMethodError
```

Comment:
I think we should do this in the `provider_a_hei?` method. It will still work here but I it's more obvious where we're expecting there might be a problem if move it there.